tests/test_intervention.py

```diff
@@ -398,17 +399,28 @@ def test_card_html_update(self):
         self.assertTrue(
             user.display_name in ae.display_for_user(user).card_html)
 
+        dt = datetime(2017, 06, 10, 20, 00, 00, 000000)
```

Comment:
putting a zero in front of an integer, means "octal" - in this case they're equal, but I think we should avoid it unless we want an octal value:
```
>>> 06 == 6
True
>>> 011 == 11
False
```